app/assets/stylesheets/content/_tables.sass

```diff
@@ -60,11 +60,21 @@ table
 
   &.workflow-table
     margin-bottom: 16px
+    border-collapse: separate
```

Comment:
Indeed I've chosen 'separate' to hide the one border below current status. The other borders can nevertheless still be seen, because every td has it's own border which is displayed then. Anyway I would keep this since it is not granted that the span is always high enough to cover this border with its overflow-x value.